Sources/Logging/LogHandler.swift

```diff
@@ -134,11 +134,6 @@ public protocol LogHandler: _SwiftLogSendableLogHandler {
              function: String,
              line: UInt)
 
-    /// SwiftLog 1.0 compatibility method. Please do _not_ implement, implement
-    /// `log(level:message:metadata:source:file:function:line:)` instead.
-    @available(*, deprecated, renamed: "log(level:message:metadata:source:file:function:line:)")
-    func log(level: Logging.Logger.Level, message: Logging.Logger.Message, metadata: Logging.Logger.Metadata?, file: String, function: String, line: UInt)
```

Comment:
As the comment states: `SwiftLog 1.0 compatibility method.` 

This method is here for compatibility reasons and we _cannot_ remove it without causing source-breaks in log handler implementations which adopted during 1.0 cycle and did not upgrade to implementing the new methods.

This was purposefully implemented this way in order to keep source compatibility, and while it is annoying to implement a new handler since you have to read the comment or note on the method, it guarantees that all 1.x libraries work well together.

We can only do such change once we do a source-breaking 2.0 release which at this point in time is not being considered.